config/attributes/clothing.json

```diff
@@ -1,33 +1,54 @@
 {
   "size": [
-    "0 (XXS)",
-    "2 (XS)",
-    "4 (S)",
-    "6 (S)",
-    "8 (M)",
-    "10 (M)",
-    "12 (L)",
-    "14 (L)",
-    "16 (XL)",
-    "18 (1X)",
-    "20 (2X)",
-    "22 (3X)"
+    "XXS",
```

Comment:
I think we use this list to populate a single "size" list that a seller can choose from.  I suppose it's okay to list all the possible options, although it might be weird to see shoe sizes when you're not trying to list a shoe.